packages/widgetbook/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ environment:
 dependencies:
   collection: ^1.15.0
   device_frame: ^1.1.0
+  flutter_platform_widgets: ^3.3.5
```

Comment:
We have a very strict policy when it comes to adding a new dependency, because it make upgrading to latest versions harder in the future.

The new dependency has to be useful for **almost all users**, which is not the case for `flutter_platform_widgets`, it's only useful for the users using it.

Reply:
I still can remove this library, to at least got the platform switch "native" flutter, what do you think ? I will probably make another addon that handle the `flutter_platform_widgets` alone in a separate package like you mention.